### PR TITLE
Using Attributes instead of manifest tag

### PR DESCRIPTION
**Pull request by TiagoBrenck**
Added the class `MsalActivity.cs` using attributes in order to replace the equivalent tag in the manifest.
I have also updated the *Readme*. Once approved, I will replicate this changes to the B2C xamarin sample and the library internal samples.

Issue link: https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/1094

### CODE REVIEW

File: 1-Basic/README.md [hunk 1]
```diff
@@ -115,25 +116,21 @@ where `[APPLICATIONID]` is the identifier you copied in step 2. Save the file.
...
+1. Open the `UserDetailsClient.Droid\MsalActivity.cs` file.
+1. Replace `[APPLICATIONID]` with the identifier you copied in step 2.
+1. Save the file.
+
+```csharp
+  [Activity]
+  [IntentFilter(new[] { Intent.ActionView },
+        Categories = new[] { Intent.CategoryBrowsable, Intent.CategoryDefault },
+        DataHost = "auth",
+        DataScheme = "msal[APPLICATIONID]")]
```
Comment (jennyf19) [inline 1-Basic/README.md]: nit: can this be client id? or change client id to application id? it's always confusing to have both.

Comment (TiagoBrenck) [reply 1-Basic/README.md]: Renamed

File: 2-With-broker/README.md [hunk 2]
```diff
@@ -124,25 +124,21 @@ where `[APPLICATIONID]` is the identifier you copied in step 2. Save the file.
...
-      </intent-filter>
-    </activity>
-      </application>
+1. Open the `UserDetailsClient.Droid\MsalActivity.cs` file.
+1. Replace `[ClientID]` with the identifier you copied in step 2.
+1. Save the file.
+
+```csharp
+  [Activity]
+  [IntentFilter(new[] { Intent.ActionView },
```
Comment (jmprieur) [inline 2-With-broker/README.md]: I think that what we should do is inherit from `IntentFilter` and add a constructor with the client ID, which would compute the DataScheme ?

Comment (TiagoBrenck) [reply 2-With-broker/README.md]: @jmprieur `IntentFilter` is a sealed class.

**Pull request merged**